app/src/main/java/org/oppia/android/app/splash/SplashActivityPresenter.kt

```diff
@@ -47,6 +49,21 @@ class SplashActivityPresenter @Inject constructor(
     activity.finish()
   }
 
+  fun loadPlatformParameters() {
+    getParameterLoadingStatus().observe(
+      activity,
+      Observer { loadSuccessful ->
+        if (!loadSuccessful) {
+          oppiaLogger.w(
+            "SplashActivity",
+            "Default PlatformParameters will be used",
```

Comment:
Can you make the log a bit more intuitive? SOmething like 'Failed to load platform parameters, defaults will be used instead.'

Reply:
Done